CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+X.Y.Z Release notes
+=============================================================
+### Breaking changes
+* None.
+
+### Enhancements
+* None
+
+### Bug fixes
+* Spurious closing of sync session (#1391).
```

Comment:
IMO this should be more explicit about what's being fixed. Something like "Avoid closing then reopening a sync session when using `Realm.open`"